### PR TITLE
SSML Refactor

### PR DESCRIPTION
**Pull request by SleepyDeveloper**
Fixed WaterCactusIntentHandler speakOutput

Signed-off-by: Justin Jeffress <sleepydeveloper@gmail.com>

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lambda/index.js [hunk 1]
```diff
@@ -804,6 +874,12 @@ const UpdateLatestInteractionResponseInterceptor = {
     }
 }
 
+const logResponseJsonResponseInterceptor = {
```
Comment (JoeMoCode) [inline lambda/index.js:877]: nit, capitalize

File: lambda/index.js [hunk 2]
```diff
@@ -898,7 +975,8 @@ exports.handler = Alexa.SkillBuilders.custom()
         NewSessionRequestInterceptor
     )
     .addResponseInterceptors(
-        UpdateLatestInteractionResponseInterceptor
+        UpdateLatestInteractionResponseInterceptor,
+        logResponseJsonResponseInterceptor
```
Comment (JoeMoCode) [inline lambda/index.js:979]: nit capitalize

File: lambda/spec/ssmlUtilSpec.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const ssmlUtil = require('../src/ssmlUtil');
+
+describe('wrapDomain', function() {
+    it('should be wrapped in the domain ssml tag', function() {
+        const speakOutput = "Hello there, I'm a cactus.";
+
+        const actual = ssmlUtil.wrapDomain(speakOutput);
+        const expected = '<amazon:domain name="long-form">Hello there, I\'m a cactus.</amazon:domain>';
```
Comment (JoeMoCode) [inline lambda/spec/ssmlUtilSpec.js:10]: nit, use the speakOutput var with `` syntax

File: lambda/spec/ssmlUtilSpec.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+});
+
+describe('wrapCactusVoice', function() {
+    it('Should wrap in output with Brian\'s voice', function() {
+
+        const speakOutput = "Hello there, I'm a cactus.";
+        const profile = require('./profiles/no-needs.json');
+
+        const actual = ssmlUtil.wrapCactusVoice(profile,speakOutput);
+        const expected = '<voice name="Brian">Hello there, I\'m a cactus.</voice>';
```
Comment (JoeMoCode) [inline lambda/spec/ssmlUtilSpec.js:24]: nit, use the `` with above speakOutput

File: lambda/src/statusUtil.js [hunk 5]
```diff
@@ -32,42 +43,61 @@ const TWO_NEEDS = [
...
+    `<prosody rate="90%">When I have a<prosody volume="x-loud"> thorny</prosody> day, I find my <prosody pitch="high">happy</prosody> place. Today my happy place is … </prosody> ${SOUND_FX.FOOTBALL}`,
+    `<prosody rate="90%">When I have a<prosody volume="x-loud"> thorny</prosody> day, I find my <prosody pitch="high">happy</prosody> place. Today my happy place is … </prosody> ${SOUND_FX.CAR_ACCELERATE}`,
+    `<prosody rate="90%">When I get sand in my <prosody volume="x-loud"> spines,</prosody> I find my <prosody pitch="high">happy</prosody> place. Today my happy place is … </prosody> ${SOUND_FX.TOILET_FLUSH}`,
+    `<prosody rate="90%">When I get sand in my <prosody volume="x-loud"> spines,</prosody> I find my <prosody pitch="high">happy</prosody> place. Today my happy place is … </prosody> ${SOUND_FX.GAME_SHOW}`,
+    `<prosody rate="90%">When I get sand in my <prosody volume="x-loud"> spines,</prosody> I find my <prosody pitch="high">happy</prosody> place. Today my happy place is … </prosody> ${SOUND_FX.CHRISTMAS_01} ${SOUND_FX.CHRISTMAS_02}`,
```
Comment (JoeMoCode) [inline lambda/src/statusUtil.js:91]: lots of ellipses chars it looks like.

Comment (SleepyDeveloper) [reply lambda/src/statusUtil.js:91]: Fixed

File: lambda/spec/ssmlUtilSpec.js (hunk 3 above)
Comment (SleepyDeveloper) [reply lambda/spec/ssmlUtilSpec.js:10]: fixed

File: lambda/spec/ssmlUtilSpec.js (hunk 4 above)
Comment (SleepyDeveloper) [reply lambda/spec/ssmlUtilSpec.js:24]: fixed

File: lambda/index.js (hunk 2 above)
Comment (SleepyDeveloper) [reply lambda/index.js:1077]: Fixed

**Pull request merged**